examples/basic.py

```diff
@@ -1,12 +1,7 @@
 import asyncio
-
```

Comment:
This shouldn't be removed, right? Because asyncio is builtin but mongox is third party.

Reply:
right